### PR TITLE
Mike.setup

### PR DESCRIPTION
**Pull request by Xadoy**
## Summary

add templates and CONTRIBUTING.md

## Related Issue

 #4 

## Alert Teammates

@mysterytony

### CODE REVIEW

**Comment by Xadoy** [comment]
@mysterytony those two lines are removed

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+## How to contribute to ExpressCPP
+
+#### **If you found a bug: **
```
Comment (mysterytony) [inline CONTRIBUTING.md]: remove `**` for all titles

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+* Ensure the bug was not already reported by searching on GitHub under [Issues](https://github.com/Lamega/ExpressCPP/issues).
+
+* If you're unable to find an open issue addressing the problem, [open a new one](https://github.com/Lamega/ExpressCPP/issues/new). Please use the given issue template to report the issue.
+
+#### **If you fixed a bug: **
+* Open a new GitHub pull request and fill out the given pull request template.
```
Comment (mysterytony) [inline CONTRIBUTING.md]: add link to the pull request template

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+* Please note that ExpressCPP is using squash merge to merge branches.
+* Before submitting, please code style guide on [Wiki](https://github.com/Lamega/ExpressCPP/wiki).
```
Comment (mysterytony) [inline CONTRIBUTING.md]: missing verb

File: ISSUE_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
+(Please label this issue as a "Bug" and choose the "Severity" label.)
```
Comment (mysterytony) [inline ISSUE_TEMPLATE.md]: use `<!-- -->` to enclose the comments

File: ISSUE_TEMPLATE.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+## Reproduce Step
+
+1. [step one]
+2. [step two]
+3. [step three...]
+
+## Environment
```
Comment (mysterytony) [inline ISSUE_TEMPLATE.md]: remove this section. we only have one environment

File: ISSUE_TEMPLATE.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+
+[what should happen]
+
+## Actual Behavior
+
+[what happened instead]
+
+## Related Branch/Pull Request
+
+* Branch: [not set/[branch name](link to the branch)]
```
Comment (mysterytony) [inline ISSUE_TEMPLATE.md]: use pound and number to generate link to a branch. for example: `#12` . same as pull request link

File: ISSUE_TEMPLATE.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+* Pull Request: [not set/#some number]
+
+---
+(Please label this issue as a "Task")
```
Comment (mysterytony) [inline ISSUE_TEMPLATE.md]: comment this

File: ISSUE_TEMPLATE.md [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+
+## Summary
+
+* [what are expected to be accomplished]
+
+## To Do
+
+[what are the detailed goals of this task]
```
Comment (mysterytony) [inline ISSUE_TEMPLATE.md]: comment this

File: ISSUE_TEMPLATE.md [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+- [ ] [goal two]
+
+## Attachment
+
+[This section is optional]
+[screen shot/help document/etc.]
+
+## Related Branch/Pull Request
+
+* Branch: [not set/[branch name](link to the branch)]
```
Comment (mysterytony) [inline ISSUE_TEMPLATE.md]: same as previous one.

File: PULL_REQUEST_TEMPLATE.md [hunk 10]
```diff
@@ -0,0 +1,11 @@
+## Summary
+
+[summary of issue/feature and your changes]
+
+## Related Issue
+
+[#some number]
+
+## Alert Teammates
```
Comment (mysterytony) [inline PULL_REQUEST_TEMPLATE.md]: change `Teammates` to `A Reviewer`

File: ISSUE_TEMPLATE.md (hunk 6 above)
Comment (Xadoy) [inline ISSUE_TEMPLATE.md]: I don't think pound can refer to a branch. It can only refer to pull requests and issues

**Pull request merged**